docs/openapi.json

```diff
@@ -384,6 +384,18 @@
                   "title": "Default workspace",
                   "value": "/usr/share"
                 },
+                "kubernetes_max_memory_limit": {
+                  "title": "Maximum allowed memory limit for Kubernetes jobs",
+                  "value": "4Gi"
+                },
+                "kubernetes_memory_limit": {
+                  "title": "Default memory limit for Kubernetes jobs",
+                  "value": "2Gi"
+                },
+                "maximum_workspace_retention_period": {
+                  "title": "Maximum retention period in days for workspace files",
+                  "value": "3"
```

Comment:
Again, 3 days would look too harsh here. Please put 3650 (=10 years) or even 36500 (=100 years) to express a really long time.